packages/flutter/lib/src/material/list_tile.dart

```diff
@@ -85,9 +88,12 @@ enum ListTileTitleAlignment {
   threeLine,
 
   /// The tops of the [ListTile.leading] and [ListTile.trailing] widgets are
-  /// placed 16 units below the top of the [ListTile.title]
-  /// if the titles' overall height is greater than 72, otherwise they're
-  /// centered relative to the [ListTile.title] and [ListTile.subtitle] widgets.
+  /// placed 16 pixels below the top of the [ListTile.title]
+  /// if the titles' overall height is greater than 72, otherwise the
+  /// [ListTile.trailing] widget is centered relative to the [ListTile.title] and
+  /// [ListTile.subtitle] widgets, and the [ListTile.leading] widget is 16 pixels
+  /// below the top of [ListTile.title], or center-aligned with [ListTile.title],
+  /// whichever makes the [ListTile.leading] closer to the top edge of [ListTile.title].
```

Comment:
Hm, what am I missing here? Isn't `tileHeight > 72` the same as saying the tile hight is greater than 72?

Reply:
Oh, I missed that one is tile and the other one is title. Probably a typo in the docs? Since nobody seems to have complained about the implementation, lets make the doc match the implementation?